### PR TITLE
Add citation section to GIS applications

### PR DESCRIPTION
**Pull request by samumantha**
Citations added directly for most applications. 
Splitted also Acknowledgement ans License sections and updated acknowledgement where needed.

Even better might be to add bibtex citation -> to be discussed

Please squash and merge.

### CODE REVIEW

File: docs/apps/cloudcompare.md [hunk 1]
```diff
@@ -43,17 +43,17 @@ CloudCompare is published under the [GNU General Public License](https://github.
...
-
-
 ## Citation
 
 ```
 CloudCompare (version 2.10.3) [GPL software]. (2021). Retrieved from <a href="http://www.cloudcompare.org/" aria-label="CloudCompare homepage">CloudCompare</a>
 ```
 
-Also cite the plugin authors, if you used a [plugin](http://www.cloudcompare.org/doc/wiki/index.php?title=Plugins).
+If you used a [plugin](http://www.cloudcompare.org/doc/wiki/index.php?title=Plugins), also cite the plugin authors.
```
Comment (samumantha) [inline docs/apps/cloudcompare.md:52]:
```suggestion
If you used a [CloudCompare plugin](http://www.cloudcompare.org/doc/wiki/index.php?title=Plugins), also cite the plugin authors.
```

File: docs/apps/force.md [hunk 2]
```diff
@@ -43,13 +43,20 @@ If working interactively, you can also start a shell inside the container with
...
 
-## License and acknowledgement
+## License 
 
-FORCE is licensed under [the GNU GPL License](https://github.com/davidfrantz/force/blob/master/LICENSE))
+FORCE is licensed under [the GNU GPL License](https://github.com/davidfrantz/force/blob/master/LICENSE).
+
+## Citation
+
+This software can be cited by citing the [appropriate publication](https://force-eo.readthedocs.io/en/latest/refs.html).
```
Comment (samumantha) [inline docs/apps/force.md:52]:
```suggestion
This software can be cited by citing the appropriate [FORCE publication](https://force-eo.readthedocs.io/en/latest/refs.html).
```

File: docs/apps/opendronemap.md [hunk 3]
```diff
@@ -70,13 +70,20 @@ rsync -r $LOCAL_SCRATCH/* $odm_dir
 ```
 
 
-## License and acknowledgement
+## License 
 
 OpenDroneMap is distributed under the GNU General Public License (GPL) version 3. Full license [here](https://github.com/OpenDroneMap/ODM/blob/master/LICENSE)
```
Comment (samumantha) [inline docs/apps/opendronemap.md:75]:
```suggestion
OpenDroneMap is distributed under the GNU General Public License (GPL) version 3. Full license from the [OpenDroneMap github repository](https://github.com/OpenDroneMap/ODM/blob/master/LICENSE)
```

File: docs/apps/otb.md [hunk 4]
```diff
@@ -81,13 +81,20 @@ otbcli_RadiometricIndices -in test_vrt.vrt -channels.nir 1 -channels.red 2 -list
 !!! note
     It is also possible to run OTB in parallel in several computing nodes [using MPI](https://www.orfeo-toolbox.org/CookBook/CliInterface.html#parallel-execution-with-mpi) but it has not been tested yet on Puhti
 
-## License and acknowledgement
+## License 
 
 Orfeo ToolBox is licensed under the Apache License, Version 2.0. The full licence can be found [here](https://github.com/orfeotoolbox/OTB/tree/develop/Copyright)
```
Comment (samumantha) [inline docs/apps/otb.md:86]:
```suggestion
Orfeo ToolBox is licensed under the Apache License, Version 2.0. The full licence can be found from the [Orfeo ToolBox github repository](https://github.com/orfeotoolbox/OTB/tree/develop/Copyright)
```

File: docs/apps/pdal.md [hunk 5]
```diff
@@ -26,13 +26,20 @@ You can test if pdal loaded successfully with following
 
 [Examples for using PDAL in Puhti](https://github.com/csc-training/geocomputing/tree/master/pdal).
 
-## License and acknowledgement
+## License 
 
 PDAL is licensed with the BSD open source license. The full licence can be found [here](https://pdal.io/copyright.html)
```
Comment (samumantha) [inline docs/apps/pdal.md:31]:
```suggestion
PDAL is licensed with the BSD open source license. The full licence can be found from the [PDAL webpage](https://pdal.io/copyright.html)
```

**Review by samumantha — COMMENTED** [review]
Accessibility improvements

**Comment by samumantha** [comment]
@ktiits , some old and new texts improved for accessibility. Some links are only URLs and not links for easier copy- paste in citation, I would leave them as is.
Any other improvements?

File: docs/apps/cloudcompare.md [hunk 6]
```diff
@@ -43,17 +43,17 @@ CloudCompare is published under the [GNU General Public License](https://github.
 
 You are free to use CloudCompare for any purpose, including commercially or for education. 
 
-
-
 ## Citation
 
 ```
 CloudCompare (version 2.10.3) [GPL software]. (2021). Retrieved from <a href="http://www.cloudcompare.org/" aria-label="CloudCompare homepage">CloudCompare</a>
```
Comment (ktiits) [inline docs/apps/cloudcompare.md:49]:
```suggestion
CloudCompare (version 2.10.3) [GPL software]. (2021). Retrieved from http://www.cloudcompare.org.
```

File: docs/apps/force.md [hunk 7]
```diff
@@ -43,13 +43,20 @@ If working interactively, you can also start a shell inside the container with
 
 We have an example of running L1 to L2 pipeline for Sentinel images in [our Github examples](https://github.com/csc-training/geocomputing/tree/master/force)
 
-## License and acknowledgement
+## License 
 
-FORCE is licensed under [the GNU GPL License](https://github.com/davidfrantz/force/blob/master/LICENSE))
+FORCE is licensed under [the GNU GPL License](https://github.com/davidfrantz/force/blob/master/LICENSE).
```
Comment (ktiits) [inline docs/apps/force.md:48]:
```suggestion
[FORCE is licensed under the GNU GPL License](https://github.com/davidfrantz/force/blob/master/LICENSE).
```

File: docs/apps/gdal.md [hunk 8]
```diff
@@ -87,13 +87,21 @@ gdalinfo /vsis3/<name_of_your_bucket>/<name_of_your_file>
 
 With large quantities of raster data (also in Allas), the most convenient method of accessing them might be [GDAL virtual rasters](../support/tutorials/gis/virtual-rasters.md). 
 
-## License and acknowledgement
+## License 
 
 GDAL is licensed under an [MIT/X style license](https://gdal.org/license.html)
```
Comment (ktiits) [inline docs/apps/gdal.md:92]:
```suggestion
[GDAL is licensed under an MIT/X style license](https://gdal.org/license.html)
```

File: docs/apps/gdal.md [hunk 9]
```diff
@@ -87,13 +87,21 @@ gdalinfo /vsis3/<name_of_your_bucket>/<name_of_your_file>
...
 
+## Citation
+
+```GDAL/OGR contributors (2022). GDAL/OGR Geospatial Data Abstraction software Library. Open Source Geospatial Foundation. URL https://gdal.org, DOI: 10.5281/zenodo.5884351 ```
```
Comment (ktiits) [inline docs/apps/gdal.md:96]:
```suggestion
```
GDAL/OGR contributors (2022). 
GDAL/OGR Geospatial Data Abstraction software Library. 
Open Source Geospatial Foundation. 
URL https://gdal.org
DOI: 10.5281/zenodo.5884351 
```
```

Comment (samumantha) [reply docs/apps/gdal.md:96]: As this is meant for copying, I would leave it as oneliner.

File: docs/apps/gdal.md [hunk 10]
```diff
@@ -87,13 +87,20 @@ gdalinfo /vsis3/<name_of_your_bucket>/<name_of_your_file>
...
 With large quantities of raster data (also in Allas), the most convenient method of accessing them might be [GDAL virtual rasters](../support/tutorials/gis/virtual-rasters.md). 
 
-## License and acknowledgement
+## License 
 
-GDAL is licensed under an [MIT/X style license](https://gdal.org/license.html)
+[GDAL is licensed under an MIT/X style license](https://gdal.org/license.html)
+
+## Citation
+
```
Comment (samumantha) [inline docs/apps/gdal.md:95]:
```suggestion
GDAL/OGR contributors (2022). 
GDAL/OGR Geospatial Data Abstraction software Library. 
Open Source Geospatial Foundation. 
URL https://gdal.org, 
DOI: 10.5281/zenodo.5884351 ```
```

File: docs/apps/opendronemap.md [hunk 11]
```diff
@@ -70,13 +70,20 @@ rsync -r $LOCAL_SCRATCH/* $odm_dir
...
 
-## License and acknowledgement
+## License 
 
-OpenDroneMap is distributed under the GNU General Public License (GPL) version 3. Full license [here](https://github.com/OpenDroneMap/ODM/blob/master/LICENSE)
+OpenDroneMap is distributed under the GNU General Public License (GPL) version 3. [Full OpenDroneMap license](https://github.com/OpenDroneMap/ODM/blob/master/LICENSE)
+
+## Citation
+
+```*OpenDroneMap Authors* ODM - A command line toolkit to generate maps, point clouds, 3D models and DEMs from drone, balloon or kite images. **OpenDroneMap/ODM GitHub Page** 2020; [https://github.com/OpenDroneMap/ODM](https://github.com/OpenDroneMap/ODM)```
```
Comment (ktiits) [inline docs/apps/opendronemap.md:79]:
```suggestion
`OpenDroneMap Authors, ODM - A command line toolkit to generate maps, point clouds, 3D models and DEMs from drone, balloon or kite images. OpenDroneMap/ODM GitHub Page 2020; https://github.com/OpenDroneMap/ODM`
```

File: docs/apps/otb.md [hunk 12]
```diff
@@ -81,13 +81,20 @@ otbcli_RadiometricIndices -in test_vrt.vrt -channels.nir 1 -channels.red 2 -list
...
 
-## License and acknowledgement
+## License 
 
-Orfeo ToolBox is licensed under the Apache License, Version 2.0. The full licence can be found [here](https://github.com/orfeotoolbox/OTB/tree/develop/Copyright)
+Orfeo ToolBox is licensed under the Apache License, Version 2.0. The full licence can be found from the [Orfeo ToolBox github repository](https://github.com/orfeotoolbox/OTB/tree/develop/Copyright)
+
+## Citation
+
+```Grizonnet, M., Michel, J., Poughon, V. et al. Orfeo ToolBox: open source processing of remote sensing images. Open geospatial data, softw. stand. 2, 15 (2017). https://doi.org/10.1186/s40965-017-0031-6```
```
Comment (ktiits) [inline docs/apps/otb.md:90]:
```suggestion
`Grizonnet, M., Michel, J., Poughon, V. et al. Orfeo ToolBox: open source processing of remote sensing images. Open geospatial data, softw. stand. 2, 15 (2017). https://doi.org/10.1186/s40965-017-0031-6`
```

File: docs/apps/otb.md [hunk 13]
```diff
@@ -81,13 +81,20 @@ otbcli_RadiometricIndices -in test_vrt.vrt -channels.nir 1 -channels.red 2 -list
 !!! note
     It is also possible to run OTB in parallel in several computing nodes [using MPI](https://www.orfeo-toolbox.org/CookBook/CliInterface.html#parallel-execution-with-mpi) but it has not been tested yet on Puhti
 
-## License and acknowledgement
+## License 
 
-Orfeo ToolBox is licensed under the Apache License, Version 2.0. The full licence can be found [here](https://github.com/orfeotoolbox/OTB/tree/develop/Copyright)
+Orfeo ToolBox is licensed under the Apache License, Version 2.0. The full licence can be found from the [Orfeo ToolBox github repository](https://github.com/orfeotoolbox/OTB/tree/develop/Copyright)
```
Comment (ktiits) [inline docs/apps/otb.md:86]:
```suggestion
Orfeo ToolBox is licensed under the Apache License, Version 2.0. [The full Orfeo ToolBox licence](https://github.com/orfeotoolbox/OTB/tree/develop/Copyright)
```

File: docs/apps/pcl.md [hunk 14]
```diff
@@ -37,15 +37,20 @@ ls /appl/soft/geo/pcl/1.9.1/bin
...
 
-## License and acknowledgement
+## License 
 
 PCL is released under the terms of the BSD license, and thus free for commercial and research use.
 
-[PCL citation](https://pointclouds.org/about/)
+## Citation
+
+```Radu Bogdan Rusu and Steve Cousins, 2011: "3D is here: Point Cloud Library (PCL)" in IEEE International Conference on Robotics and Automation (ICRA)```
```
Comment (ktiits) [inline docs/apps/pcl.md:46]:
```suggestion
`Radu Bogdan Rusu and Steve Cousins, 2011: "3D is here: Point Cloud Library (PCL)" in IEEE International Conference on Robotics and Automation (ICRA)`
```

File: docs/apps/pdal.md [hunk 15]
```diff
@@ -26,13 +26,20 @@ You can test if pdal loaded successfully with following
 
 [Examples for using PDAL in Puhti](https://github.com/csc-training/geocomputing/tree/master/pdal).
 
-## License and acknowledgement
+## License 
 
-PDAL is licensed with the BSD open source license. The full licence can be found [here](https://pdal.io/copyright.html)
+PDAL is licensed with the BSD open source license. The full licence can be found from the [PDAL webpage](https://pdal.io/copyright.html)
```
Comment (ktiits) [inline docs/apps/pdal.md:31]:
```suggestion
PDAL is licensed with the BSD open source license. [The full PDAL licence](https://pdal.io/copyright.html)
```

File: docs/apps/qgis.md [hunk 16]
```diff
@@ -57,21 +57,24 @@ It is also possible to access QGIS functionalities from Python without an graphi
 
 QGIS can __read__ files directly from Allas, either using S3 or SWIFT API. Before starting QGIS set up the connection as described in on [GDAL](gdal.md) page.
 
-## License and acknowledgement
+## License 
 
 QGIS is licensed under the GNU General Public License. QGIS is an official project of the Open Source Geospatial Foundation (OSGeo).
```
Comment (ktiits) [inline docs/apps/qgis.md:62]:
```suggestion
QGIS is licensed under the GNU General Public License.
```

File: docs/apps/saga-gis.md [hunk 17]
```diff
@@ -76,10 +76,16 @@ saga_gui
...
 
 
-## License and acknowledgement
+## License
 
 SAGA GIS is published under the [GPL](http://www.gnu.org/licenses/gpl.html) license. 
 
+## Citation
+
+```Conrad, O., Bechtel, B., Bock, M., Dietrich, H., Fischer, E., Gerlitz, L., Wehberg, J., Wichmann, V., and Böhner, J. (2015): System for Automated Geoscientific Analyses (SAGA) v. 2.1.4, Geosci. Model Dev., 8, 1991-2007, doi:10.5194/gmd-8-1991-2015.  ```
```
Comment (ktiits) [inline docs/apps/saga-gis.md:85]:
```suggestion
`Conrad, O., Bechtel, B., Bock, M., Dietrich, H., Fischer, E., Gerlitz, L., Wehberg, J., Wichmann, V., and Böhner, J. (2015): System for Automated Geoscientific Analyses (SAGA) v. 2.1.4, Geosci. Model Dev., 8, 1991-2007, doi:10.5194/gmd-8-1991-2015.  `
```

File: docs/apps/sen2cor.md [hunk 18]
```diff
@@ -36,13 +36,20 @@ Further configuration of `L2A_Process` can be done via the `L2A_GIPP.xml` file.
...
 Here you can adjust it to your needs using any editor and add it to the call: `L2A_Process --GIP_L2A $HOME/sen2cor/2.10/cfg/L2A_GIPP.xml <other arguments>` .
 
-## License and acknowledgement
+## License 
 
 Sen2Cor is published under the [GPL-3](https://www.gnu.org/licenses/gpl.html) license.
 
+## Citation
+
+``` M. Main-Knorn, B. Pflug, J. Louis, V. Debaecker, U. Müller-Wilm, F. Gascon, "Sen2Cor for Sentinel-2", Proc. SPIE 10427, Image and Signal Processing for Remote Sensing XXIII, 1042704 (2017)```
```
Comment (ktiits) [inline docs/apps/sen2cor.md:45]:
```suggestion
` M. Main-Knorn, B. Pflug, J. Louis, V. Debaecker, U. Müller-Wilm, F. Gascon, "Sen2Cor for Sentinel-2", Proc. SPIE 10427, Image and Signal Processing for Remote Sensing XXIII, 1042704 (2017)`
```

File: docs/apps/snap.md [hunk 19]
```diff
@@ -151,13 +151,20 @@ apptainer_wrapper exec pip <NEW-PACKAGE-NAME> --user
 
 SNAP minor and module updates are stored in the `$HOME/.snap` directory. This means that all minor updates need to be installed by the user. You can either do this in SNAP Desktop by following the instructions in the pop-up at start up, or for SNAP 9 by running `source update_snap` after loading the SNAP module in the terminal.
 
-## License and acknowledgement
+## License
 
 All SNAP software is published under the [GPL-3](https://www.gnu.org/licenses/gpl.html) license and its sources are available on [GitHub](https://github.com/senbox-org/).
```
Comment (ktiits) [inline docs/apps/snap.md:156]:
```suggestion
All SNAP software is published under the [GPL-3](https://www.gnu.org/licenses/gpl.html) license.
```

File: docs/apps/whiteboxtools.md [hunk 20]
```diff
@@ -52,13 +52,20 @@ module load whiteboxtools
 whitebox_tools -r=Hillshade -v -i=/appl/data/geo/mml/dem10m/2019/M3/M34/M3444.tif -o=test_hillshade.tif --azimuth=315.0 --altitude=30.0
 ```
 
-## License and acknowledgement
+## License 
 
 The WhiteboxTools open-core platform is distributed under the [MIT license](https://www.whiteboxgeo.com/manual/wbt_book/license.html)
```
Comment (ktiits) [inline docs/apps/whiteboxtools.md:57]:
```suggestion
The WhiteboxTools open-core platform is distributed under the MIT license. [The full WhiteboxTools license](https://www.whiteboxgeo.com/manual/wbt_book/license.html)
```

File: docs/apps/whiteboxtools.md [hunk 21]
```diff
@@ -52,13 +52,20 @@ module load whiteboxtools
...
 
+## Citation
+
+```Lindsay, J.B. (2014) The Whitebox Geospatial Analysis Tools project and open-access GIS, Proceedings of the GIS Research UK 22nd Annual Conference, The University of Glasgow, 16-18 April, DOI: 10.13140/RG.2.1.1010.8962.```
```
Comment (ktiits) [inline docs/apps/whiteboxtools.md:61]:
```suggestion
`Lindsay, J.B. (2014) The Whitebox Geospatial Analysis Tools project and open-access GIS, Proceedings of the GIS Research UK 22nd Annual Conference, The University of Glasgow, 16-18 April, DOI: 10.13140/RG.2.1.1010.8962.`
```

File: docs/apps/zonation.md [hunk 22]
```diff
@@ -64,13 +64,20 @@ cd /scratch/<your_project>/manual_and_example_setups/example\ setups\ and\ data/
...
 ```
 
-## License and acknowledgement
+## License 
 
 Zonation 5 is distributed as is, freely under [GNU General Public License (GPL) version 3 (#GNUGPL) (#GNUGPLv3) license.](https://www.gnu.org/licenses/gpl-3.0.html)
 
+## Citation
+
+```Moilanen, A., Lehtinen, P., Kohonen, I., Virtanen, E., Jalkanen, J. and Kujala, H. 2022. Novel methods for spatial prioritization with applications in conservation, land use planning and ecological impact avoidance. Methods in Ecology and Evolution```
```
Comment (ktiits) [inline docs/apps/zonation.md:73]:
```suggestion
`Moilanen, A., Lehtinen, P., Kohonen, I., Virtanen, E., Jalkanen, J. and Kujala, H. 2022. Novel methods for spatial prioritization with applications in conservation, land use planning and ecological impact avoidance. Methods in Ecology and Evolution`
```

**Review by ktiits — APPROVED** [review]

**Pull request merged**